### PR TITLE
DEVENGAGE-603 Add DID pool creation and DID assignment

### PR DESCRIPTION
**Pull request by PrinceMerluza**
DEVENGAGE-603 
Add DID pool creation and DID assignment

+ Added DID Pool creation scripts for PowerShell and *nix-shells

Waiting on new CLI version that includes DID assignment for IVRs.

### CODE REVIEW

File: telephony/powershell/New-DIDPool.ps1 [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    endPhoneNumber = "13175550099"      # Ending phone number for the range. Must be in E-164 format
+} | ConvertTo-Json |
+    gc.exe edges didpools create        # gc cli command to create the DID pool
+
+
+# Create a DID pool from a pre-defined JSON file.
+# For this method, you should have a JSON file prepared with the following format:
+<#
+    {
+        "description": "Twilio",
```
Comment (carnellj) [inline telephony/powershell/New-DIDPool.ps1:26]: Hey Prince can you change the name from Twilio to something else.  While it doesn't matter from an execution perspective I would rather not use their names in a code example

**Review by carnellj — COMMENTED** [review]
Quick question.

Do you have an example in powershell of doing the assignment of a DID to a call flow.  This involves retrieving the record using /architect/ivr/{ivr id} and then updating the DNIS array with a new value.

**Comment by PrinceMerluza** [comment]
> Quick question.
> 
> Do you have an example in powershell of doing the assignment of a DID to a call flow. This involves retrieving the record using /architect/ivr/{ivr id} and then updating the DNIS array with a new value.

I was waiting on the new CLI with the IVR feature, but didn't notice it was already updated. I've just added the sample for DID assignment.

File: telephony/powershell/New-DIDPool.ps1 (hunk 1 above)
Comment (PrinceMerluza) [reply telephony/powershell/New-DIDPool.ps1]: + Changed to "Service Provider X"

**Review by carnellj — APPROVED** [review]

**Pull request merged**